memtier_benchmark.cpp

```diff
@@ -751,6 +790,21 @@ static int config_parse_args(int argc, char *argv[], struct benchmark_config *cf
                     }
                     break;
                 }
+                case o_tls:
```

Comment:
I'm understanding that having that code without define USE_TLS do nothing, but since we already wrap every piece of code that related to TLS with USE_TLS maybe it will be more clearly to wrap this section too (this come with my below comment about wrapping the variables declaration too)